src/Domain/ValetToken/ValetToken.ts

```diff
@@ -0,0 +1,33 @@
+import { DateString } from '../Date/Date'
+import { Operation } from '../Operation/Operation'
+import { Resource } from '../Resource/Resource'
+import { Uuid } from '../Uuid/Uuid'
+
+
+export type ValetToken = string
+export type ValetKeySignature = string
+
+export type ValetStructure = {
```

Comment:
please apply hints mentioned above on this whole file

Reply:
I assume this is the same point as in here (the one x per file rule): https://github.com/standardnotes/files/pull/3#discussion_r598782928